sky/shell/platform/android/library_loader.cc

```diff
@@ -35,15 +35,22 @@ bool RegisterJNI(JNIEnv* env) {
                                arraysize(kSkyRegisteredMethods));
 }
 
+bool InitJNI() {
+  return sky::shell::InitJniSkyMain();
```

Comment:
Why bounce through sky/shell?  Seems like you could go to blink::DartJni::InitJni directly.